### DISCUSSION TITLE
Enhancement: Support dify.ai endpoint

### DISCUSSION THREAD

**Discussion by shota1998** [danny-avila/LibreChat#2830] [Feature Requests & Suggestions]
### What features would you like to see added?

Users can choose dify.ai as an endpoint.

### More details

Dify is an open-source  LLM application development platform.
https://dify.ai/

With dify.ai, it is possible to build the backend for LLM  applications.
Users can use librechat as a frontend, and use difty.ai as a backend.

### Which components are impacted by your request?

Endpoints

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Thanks! Will get on this

**Reply by Sn0wfir3** [reply]
Hey,
How are we doing on the dify integration ?. Would be awesome!

**Comment by rnavarroz** [comment]
is there any plan for the dify integration?

**Comment by dnna** [comment]
+1

**Comment by danny-avila** [comment]
I'm not planning on this. If they expose OpenAI-compatible endpoints, it can be supported already: 
https://www.librechat.ai/docs/quick_start/custom_endpoints
